scalyr_agent/copying_manager.py

```diff
@@ -289,24 +289,24 @@ def __path_in_globs( self, path, path_globs ):
 
         return None
 
-    def add_log_config( self, monitor, log_config ):
+    def add_log_config( self, monitor_name, log_config ):
```

Comment:
I see at least 3 calls to `add_log_config where the first param is a monitor, not a monitor_name.

![image](https://user-images.githubusercontent.com/48775713/59236832-afb4ac80-8bac-11e9-8a83-1bf94938da08.png)

Reply:
I see only 1 call in that list that was passing in the monitor rather than the monitor name (this is now fixed), unless I'm missing something, the other 2 are function definitions where self is always the first parameter.

Reply:
You're right, LGTM. The tests are still failing because of the bug in option-b/master I presume?